.github/workflows/push_pr.yml

```diff
@@ -0,0 +1,74 @@
+on:
+  release:
+    types: [ created ]
+
+name: on-creating-new-tag
```

Comment:
the name isn't accurate, the trigger is actually the creation of a release, not a tag